### DISCUSSION TITLE
Napi forgalmi jelentés információk

### DISCUSSION THREAD

**Discussion by QSoft-develop** [nav-gov-hu/Online-Cash-Register-Logfile#47] [Általános]
A következő cikk jelent meg:
https://nav.gov.hu/ado/egyeb/jelentosen-megkonnyiti-a-konyvelok-dolgat-a-nav-azzal-hogy-letolthetok-az-online-penztargepek-adatai

Áttanulmányoztam az ott hivatkozott dokumentációt ( https://github.com/nav-gov-hu/Online-Cash-Register-Logfile/tree/master/docs )
/ mellesleg nincs rajta oldalszámozás / 
Megállapítottam, hogy a letölthetőség -ha jól értem-  a beküldött naplóra vonatkozik. Ezt a könyvelők leginkább semmire nem tudják használni.
A könyvelés számára <b>napi forgalmi jelentés</b>-re lenne szükség. Viszont erre vonatkozó API hívást nem találtam.
A kérdésem: ki lehet-e nyerni (és hogyan) a letölthető adatokból a napi forgalmi jelentés információkat (azaz a kasszagép zárásokat).

Hasznos lenne egy olyan funkció is, hogy adott adó-törzsszámon milyen AP-kódú pénztárgépek vannak. Úgy láttam, ezt már más is hiányolta.

Köszönettel...

Kiss Károly

**Comment by renced42** [comment]
Kedves @QSoft-develop 

A dokumentumra vonatkozó jelzést köszönjük, frissítettük is a repót.
A naplóállományokban minden olyan bizonylat, amit a pénztárgép ki tud nyomtatni benne van, többek között a napi forgalmi jelentés is.
Erre tehát nincs API hívás ezt a letöltendő naplóállományokból kell kiolvasni. XSD-ben a P11-es Type lesz az.

**Comment by blaky** [comment]
A visszaadott allomanyokban a ROWS.NFN node-okat keresd, tobb infot a jogszabalyban a 6.3-as bekezdes alatt talalsz: https://net.jogtar.hu/jogszabaly?docid=a1300048.ngm
